### PR TITLE
release/public-v1: bugfixes for no_nsst suites

### PR DESCRIPTION
**Pull request by climbfuji**
Update .gitmodules and submodule pointer for fv3atm for code review and testing.

Associated PRs:

https://github.com/NOAA-EMC/fv3atm/pull/110
https://github.com/ufs-community/ufs-weather-model/pull/118

For regression testing information, see below.

### CODE REVIEW

**Comment by ligiabernardet** [comment]
Thanks for fixing the CCPP version number. The title of this PR alarmed me, but I see that the bug fix does not change answers, so that is a relief.

**Comment by climbfuji** [comment]
Regression testing on hera.intel, cheyenne.intel, cheyenne.gnu against existing baseline: all tests pass, regression test logs updated in PR.

[rt_cheyenne_intel.log](https://github.com/ufs-community/ufs-weather-model/files/4593193/rt_cheyenne_intel.log)
[rt_cheyenne_gnu.log](https://github.com/ufs-community/ufs-weather-model/files/4593194/rt_cheyenne_gnu.log)
[rt_hera_intel.log](https://github.com/ufs-community/ufs-weather-model/files/4593195/rt_hera_intel.log)

**Review by junwang-noaa — APPROVED** [review]

**Comment by climbfuji** [comment]
@DusanJovic-NOAA can you please have a look at this PR and https://github.com/NOAA-EMC/fv3atm/pull/110? They need to be merged into the public release branch for the 1.1.0 release. Tiny bug fix of the version number in the CCPP suite definition files for the no_nsst SDFs. Jun already reviewed and approved both. Thanks ...

**Review by DusanJovic-NOAA — APPROVED** [review]

**Comment by climbfuji** [comment]
Hmmm I think this was the wrong one ... we should have done fv3atm first?

**Comment by junwang-noaa** [comment]
Sorry, I thought I had committed fv3atm, will revert the commit.

File: .gitmodules [hunk 1]
```diff
@@ -1,7 +1,9 @@
 [submodule "FV3"]
 	path = FV3
-	url = https://github.com/NOAA-EMC/fv3atm
-	branch = release/public-v1
+	#url = https://github.com/NOAA-EMC/fv3atm
+	#branch = release/public-v1
+	url = https://github.com/climbfuji/fv3atm
```
Comment (climbfuji) [inline .gitmodules:5]: @junwang-noaa we accidentally merged this temporary change to .gitmodules - need to fix and possibly correct submodule pointer for fv3atm.

Comment (climbfuji) [reply .gitmodules:5]: Please ignore. This was fixed in a follow-up PR.

**Pull request merged**